oat-sa/tao-core-shared-libs-fe
# feat: add sourcedialog plugin

related to https://oat-sa.atlassian.net/browse/PISA25-364

### Description 

Updates ckeditor build with enable Source dialog plugin https://ckeditor.com/cke4/addon/sourcedialog

### How to test

`npm run test`

check with installed version of tao or on deployed kitchen environment (see environment field of the ticket)
